Decisions.MongoDB/BaseReplaceStep.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using DecisionsFramework;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.Design.Flow.Mapping;
+using DecisionsFramework.Design.Flow.Mapping.InputImpl;
+using DecisionsFramework.Design.Properties;
+
+namespace Decisions.MongoDB
+{
+    [Writable]
+    public abstract class BaseReplaceStep : BaseMongoDBStep, ISyncStep, IDataConsumer, IDefaultInputMappingStep
```

Comment:
Let's remove this class since we're removing the `ReplaceBulkDocumentStep` and go back to how it was before.